File: java/Tayangan.java
public class Tayangan {
    // atribut
    private String id;
    private String judul;
    private int durasi;
    private String jamTayang;
    private double hargaTiket;

    // konstruktor kosong
    public Tayangan (){
    }

    // konstruktor
    public Tayangan (String id, String judul, int durasi, String jamTayang, double hargaTiket){
        this.id = id;
        this.judul = judul;
        this.durasi = durasi;
        this.jamTayang = jamTayang;
        this.hargaTiket = hargaTiket;
    }

    // getter
    public String getId(){
        return id;
    }
    public String getJudul(){
        return judul;
    }
    public int getDurasi(){
        return durasi;
    }
    public String getJamTayang(){
        return jamTayang;
    }
    public double getHargaTiket(){
        return hargaTiket;
    }

    // setter
    public void setId(String id){
        this.id = id;
    }
    public void setJudul(String judul){
        this.judul = judul;
    }
    public void setDurasi(int durasi){
        this.durasi = durasi;
    }
    public void setJamTayang(String jamTayang){
        this.jamTayang = jamTayang;
    }
    public void setHargaTiket(double hargaTiket){
        this.hargaTiket = hargaTiket;
    }
}

File: cpp/Tayangan.cpp
#include <iostream>
#include <string>
using namespace std;

class Tayangan{
private:
    // atribut
    string id;
    string judul;
    int durasi;
    string jamTayang;
    double hargaTiket;

public:
    // konstruktor kosong
    Tayangan(){
    }

    // konstruktor
    Tayangan(string id, string judul, int durasi, string jamTayang, double hargaTiket){
        this->id = id;
        this->judul = judul;
        this->durasi = durasi;
        this->jamTayang = jamTayang;
        this->hargaTiket = hargaTiket;
    }

    // getter
    string getId(){
        return id;
    }
    string getJudul(){
        return judul;
    }
    int getDurasi(){
        return durasi;
    }
    string getJamTayang(){
        return jamTayang;
    }
    double getHargaTiket(){
        return hargaTiket;
    }
    
    // setter
    void setId(string id){
        this->id = id;
    }
    void setJudul(string judul){
        this->judul = judul;
    }
    void setDurasi(int durasi){
        this->durasi = durasi;
    }
    void setJamTayang(string jamTayang){
        this->jamTayang = jamTayang;
    }
    void setHargaTiket(double hargaTiket){
        this->hargaTiket = hargaTiket;
    }

    // destruktor
    ~Tayangan(){
    }
};